framework/arcane-framework/src/main/scala/services/mssql/MsSqlDataProvider.scala

```diff
@@ -36,13 +36,16 @@ given HasVersion[VersionedBatch] with
  * A data provider that reads the changes from the Microsoft SQL Server.
  * @param msSqlConnection The connection to the Microsoft SQL Server.
  */
-class MsSqlDataProvider(msSqlConnection: MsSqlConnection) extends VersionedDataProvider[Long, VersionedBatch] {
+class MsSqlDataProvider(msSqlConnection: MsSqlConnection) extends VersionedDataProvider[Long, VersionedBatch]
+  with BackfillDataProvider:
+  
   implicit val dataQueryRunner: QueryRunner[LazyQueryResult.OutputType, LazyQueryResult] = QueryRunner()
   implicit val versionQueryRunner: QueryRunner[Option[Long], ScalarQueryResult[Long]] = QueryRunner()
 
   override def requestChanges(previousVersion: Option[Long], lookBackInterval: Duration): Task[VersionedBatch] =
     ZIO.fromFuture(_ => msSqlConnection.getChanges(previousVersion, lookBackInterval))
-}
+    
+  override def provideData: Task[BackFillBatch] = ZIO.fromFuture(_ => msSqlConnection.backfill)
```

Comment:
Minor: consistent naming - if this comes from Backfill provider, call method requestBackfill (in line with requestChanges)

Reply:
Fixed